### ISSUE TITLE
Perpetual changes to apply when `block_http_code` is not specified

### ISSUE THREAD

**Issue by sjparkinson** [signalsciences/terraform-provider-sigsci#161]
Looks like https://github.com/signalsciences/terraform-provider-sigsci/pull/156 causes changes to apply to always appear if a site is defined without specifying `block_http_code` in the `sigsci_site` resource.

This started after upgrading to v1.2.21.

```hcl
resource "sigsci_site" "this" {
  short_name   = "production"
  display_name = "Production"
  agent_level  = "block"
}
```

```hcl
~ resource "sigsci_site" "production" {
  - block_http_code        = 406 -> null
    id                     = "production"
    # (5 unchanged attributes hidden)
}
```

### References

* https://github.com/signalsciences/terraform-provider-sigsci/pull/156

**Comment by daniel-corbett** [comment]
Hi @sjparkinson - thanks for the report! You are 100% right that #156 was the culprit. I have submitted a PR to fix this in #162 

Thanks again for the report and apologies for the issues caused here.

**Comment by shawnps** [comment]
@sjparkinson hopefully https://github.com/signalsciences/terraform-provider-sigsci/pull/162 fixed this -- if not, please reopen this issue. Thank you!
